spec/models/ont/instrument_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
```

Comment:
maybe worth testing the max flowcells works by creating an instrument and trying to assign more flowcells than allowed? Not sure if thats tested elsewhere